homeassistant/components/broadlink/switch.py

```diff
@@ -23,15 +23,14 @@
 from homeassistant.helpers.restore_state import RestoreEntity
 from homeassistant.util import Throttle, slugify
 
-from . import async_setup_service, data_packet
+from . import async_setup_service, data_packet, hostname, mac_address
+from .const import DEFAULT_PORT, DEFAULT_RETRY, DEFAULT_TIMEOUT
 
 _LOGGER = logging.getLogger(__name__)
 
 TIME_BETWEEN_UPDATES = timedelta(seconds=5)
 
 DEFAULT_NAME = "Broadlink switch"
-DEFAULT_TIMEOUT = 10
-DEFAULT_RETRY = 2
 CONF_SLOTS = "slots"
```

Comment:
I would suggest to move all constants to const.py to keep them all in one place.